frontend/public/locales/en/translation.json

```diff
@@ -2115,7 +2116,8 @@
     "submariner.install.form.gateways.labelHelp": "The number of worker nodes  that will be used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA will be enabled automatically. (default 1)",
     "submariner.install.form.gateways.placeholder": "Enter gateway count",
     "submariner.install.form.instancetype": "Instance type",
```

Comment:
_Enter gateway count_ has a prompt but _Instance type_ doesn't. Do they both need a prompt?

Reply:
Sorry , I am not clear about this, I don't see a prompt here

Reply:
The prompt here is "Enter"

Reply:
oh ok, I added in the prompt in instance type as well,